src/utils/getPreparedPeople.ts

```diff
@@ -0,0 +1,55 @@
+import { URLSearchParams } from 'url';
```

Comment:
The import statement `import { URLSearchParams } from 'url';` is incorrect for a browser environment. You should use `import { useSearchParams } from 'react-router-dom';` in the component where you need to access search parameters, and pass them to this utility function as you are already doing.